routes/index.js

```diff
@@ -9,7 +9,7 @@ const express       = require('express'),
 const urlencodedParser = bodyParser.urlencoded({extended: false});
 /* GET home page. */
 router.get('/t1', function(req, res, next) {
-  if(req.cookies['auth0']){
+  if(dataFunc.searchForCookie(req.cookies['auth0'])){
     res.send('1');
```

Comment:
is that really necessary to make in if statement and then send the number that the function will return. just return function value